### PR TITLE
Bump NodeJS from v16 to v18

### PR DESCRIPTION
**Pull request by queengooborg**
This PR bumps the NodeJS version within GitHub Actions from v16 and v18 and adds a corresponding `.nvmrc`, following an MDN-wide upgrade.

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+v18
```
Comment (caugner) [inline .nvmrc]: Trailing newline:

```suggestion
v18

```

**Review by caugner — CHANGES_REQUESTED** [review]

**Review by caugner — CHANGES_REQUESTED** [review]
And let's also update the `engines` field in `package.json`:

https://github.com/mdn/interactive-examples/blob/cdece76142518dc33555dbb64821fe8f86426646/package.json#L36

**Review by caugner — APPROVED** [review]

**Comment by queengooborg** [comment]
Note: the test is failing because it is using the version from `main`, not the version from this PR.

**Comment by caugner** [comment]
> Note: the test is failing because it is using the version from `main`, not the version from this PR.

I guess that's because we use the `pull_request` event, not the `pull_request_target` event, right?

https://github.com/mdn/interactive-examples/blob/a7196ee4a1af4efeb6616a9be6741ebf74670fa6/.github/workflows/main.yml#L3-L4

**Review by caugner — APPROVED** [review]

**Pull request merged**